.eslintrc.json

```diff
@@ -19,14 +19,15 @@
     "prettier/@typescript-eslint"
   ],
   "rules": {
-    "@typescript-eslint/no-floating-promises": "error",
     "@typescript-eslint/camelcase": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-floating-promises": "error",
```

Comment:
Did these changes end up here as a general upgrade, or are these specific to the repository changes?

Reply:
Well, the rules are now sorted which accounts for most of the changes, but I did disable `no-useless-constructor` in favor of the `@typescript-eslint/no-useless-constructor`